docs/index.bs

```diff
@@ -222,6 +222,13 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
 
     A <dfn export id="dfn-service-worker-client" for="">service worker client</dfn> is an [=environment=].
 
+    A [=/service worker client=] has an associated <dfn export>discarded flag</dfn>. It is initially unset.
+
+   This specification defines the following [=environment discarding steps=] for a [=/service worker client=] |client|:
+        1. Set |client|'s [=discarded flag=].
+
+    Note: Implementations can essentially discard clients whose [=discarded flag=] is set.
```

Comment:
Nit: lose "essentially". Also, is this defined somewhere?

Reply:
Removed essentially. Does "this" mean  [=environment discarding steps=]? They are defined in HTML: https://html.spec.whatwg.org/multipage/webappapis.html#environment-discarding-steps